Workflow/Sources/WorkflowLogger.swift

```diff
@@ -88,41 +88,4 @@ final class WorkflowLogger {
             os_signpost(.end, log: .workflow, name: "Render", signpostID: signpostID)
         }
     }
-
-    // MARK: - Workers
-
-    static func logWorkerStartedRunning<WorkerType>(ref: AnyObject, workerType: WorkerType.Type) {
```

Comment:
Should these be moved to the other pod?

If we decide to remove them altogether we should also remove the definition of the `worker` log handle, at the top.

Reply:
Since `Workers` are `Workflows` now, we should be able to get all the logs through those. 

I did notice we don't logs for `SideEffects` though.

Reply:
Oh yeah, it would be nice to log the start and end of side-effects for sure

Reply:
We'll have to figure out how to filter `Worker` logs from the `Workflow` ones, so we don't end up double-logging.